pkg/rtcconfig/config.go

```diff
@@ -68,3 +84,69 @@ func (conf *RTCConfig) Validate(development bool) error {
 
 	return nil
 }
+
+type PortRange struct {
+	Start int
+	End   int
+}
+
+func (r PortRange) MarshalYAML() (interface{}, error) {
+	if r.End == 0 {
+		return r.Start, nil
+	}
+
+	if r.End <= r.Start {
+		return nil, fmt.Errorf("end port %d must be greater than start port %d", r.End, r.Start)
+	}
+	return fmt.Sprintf("%d-%d", r.Start, r.End), nil
+}
+
+func (r *PortRange) UnmarshalYAML(value *yaml.Node) error {
+	if value.Value == "" {
+		return nil
+	}
+	if strings.Contains(value.Value, "-") {
```

Comment:
Is this due to some backwards compatibility, i. e. doing integer or string? Can this type be straight YAML rather than coverting to string?

Reply:
This struct is copied from cloud.relayConfig, the compatibility might come from there